tests/blueprints/integration/slack_test.py

```diff
@@ -0,0 +1,173 @@
+import pytest
+
+from busy_beaver.blueprints.integration.slack import (
+    ACCOUNT_ALREADY_ASSOCIATED,
+    UNKNOWN_COMMAND,
+    VERIFY_ACCOUNT,
+    reply_to_user_with_github_login_link,
+)
+from busy_beaver.models import User
+
+MODULE_TO_TEST = "busy_beaver.blueprints.integration.slack"
+
+
+@pytest.fixture
+def patcher(monkeypatch):
+    """Helper to patch in the correct spot"""
+
+    def _patcher(namespace, replacement_object):
+        namespace_to_patch = f"{MODULE_TO_TEST}.{namespace}"
+        monkeypatch.setattr(namespace_to_patch, replacement_object)
+        return replacement_object
+
+    yield _patcher
```

Comment:
This is something new I'm trying out. Still figuring it out. Lemme know if you like using it.

Basic idea is that it patches the exact namespace you want, which is always one of my biggest gripes with `mock.patch()`

Reply:
I'm not sure I understand how this one works, I'll have to get some hands on with it to see how it works